### PR TITLE
My First Pull Request - Daniel Spray

### PR DESCRIPTION
**Pull request by dfspray**

### CODE REVIEW

**Comment by dfspray** [comment]
I made quite a few mistakes assembling the folder structure, so most of the commits are adding and removing folders to match the class' structure... sorry!

File: students/Daniel_Spray/Lesson02/FizzBuzzTest.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+def FizzBuzzTest():
```
Comment (charlesrobison) [inline students/Daniel_Spray/Lesson02/FizzBuzzTest.py]: Function names in Python should be snake_case: fizz_buzz_test for example

File: students/Daniel_Spray/Lesson02/FizzBuzzTest.py [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+    for n in range(1,100):
+        if n%3==0 and n%5==0:
+            print("FizzBuzz")
+        elif n%3==0:
+            print("Fizz")
+        elif n%5==0:
+            print("Buzz")
+        else:
+            print(n)
```
Comment (charlesrobison) [inline students/Daniel_Spray/Lesson02/FizzBuzzTest.py]: Works well.  This should be 1-100 inclusive - this is going to 99.

File: students/Daniel_Spray/Lesson02/Grid_Printer.py [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+    print(plus+pluses*(minus*minuses+plus))
+    while m<pluses:
+        lowergrid(pluses,minuses,plus,minus)
+        m+=1
+def lowergrid(pluses,minuses,plus,minus):
+    n=0
+    while n<minuses:    
+        print("|"+pluses*(" "*minuses+"|"))
+        n+=1
+    print(plus+pluses*(minus*minuses+plus))
```
Comment (charlesrobison) [inline students/Daniel_Spray/Lesson02/Grid_Printer.py]: Not exactly sure what this is supposed to output...

File: students/Daniel_Spray/Lesson02/Grid_Printer.py [hunk 4]
```diff
@@ -0,0 +1,25 @@
+def print_grid0():
+    print_grid2(2,4)
```
Comment (charlesrobison) [inline students/Daniel_Spray/Lesson02/Grid_Printer.py]: Add a blank row between functions for readability.

File: students/Daniel_Spray/Lesson02/series.py [hunk 5]
```diff
@@ -0,0 +1,27 @@
+def fibonacci(n):
+    l = [0,1]
+    commonfunction(l,n)
+    return l
```
Comment (charlesrobison) [inline students/Daniel_Spray/Lesson02/series.py]: Add a blank row between functions.

File: students/Daniel_Spray/Lesson02/series.py [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+    l = []
+    l.append(a)
+    l.append(b)
+    commonfunction(l,n)
+    return l
+
+assert fibonacci(5)==[0,1,1,2,3],"The Program Returned an Incorrect Sequence for the Fibonacci Function Test"
+assert lucas(5)==[2,1,3,4,7],"The Program Returned an Incorrect Sequence for the Lucas Function Test"
+assert sum_series(3)==fibonacci(3),"The Default Output for sum_series was NOT Fibonacci"
+assert sum_series(3,2,1)==lucas(3),"The Lucas Configured sum_series Returned an Incorrect Sequence"
```
Comment (charlesrobison) [inline students/Daniel_Spray/Lesson02/series.py]: Looks good - nice assert statements

**Pull request merged**